content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+title: 'Join us at the Contributor Summit in Shanghai'
+date: 2019-06-11
+---
+
+**Author**: Josh Berkus (Red Hat)
+
+![Picture of contributor panel at 2018 Shanghai contributor summit.  Photo by Josh Berkus, licensed CC-BY 4.0](/images/blog/2019-
+06-11-contributor-summit-shanghai/panel.png)
+
+For the second year, we will have [a Contributor Summit event](https://www.lfasiallc.com/events/contributors-summit-china-2019/) the day before [KubeCon China](https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2019/) in Shanghai. If you already contribute to Kubernetes or would like to contribute, please consider attending and [register](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/). The Summit will be held June 24th, at the Shanghai Expo Center (the same location where KubeCon will take place), and will include a Current Contributor Day as well as the New Contributor Workshop and the Documentation Sprints.
```

Comment:
```suggestion
For the second year, we will have [a Contributor Summit event](https://www.lfasiallc.com/events/contributors-summit-china-2019/) the day before [KubeCon China](https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2019/) in Shanghai. If you have already contributed to Kubernetes or would like to contribute, please consider attending and [register](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/). The Summit will be held on June 24th, at the Shanghai Expo Center (the same exact location where KubeCon will take place), and will include a Current Contributor Day, the New Contributor Workshop, and the Documentation Sprints.
```

Reply:
Same as above.  The original sentence is more "natural".